cmd/debos/debos.go

```diff
@@ -29,21 +37,32 @@ func main() {
 		TemplateVars  map[string]string `short:"t" long:"template-var" description:"Template variables"`
 	}
 
+	// Allow to run all deffer calls prior to os.Exit()
+	defer func() {
+		switch p := recover(); p {
+		case code0, code1:
+			os.Exit(p.(int))
```

Comment:
I found this use of `panic()` even in successful cases a bit surprising: is this somehting idiomatic in golang?

Reply:
In fact no. Rethinked panic() usage and use different approach.